crates/compiler/gen_dev/src/generic64/aarch64.rs

```diff
@@ -1477,6 +1477,19 @@ fn sub_reg64_reg64_imm12(
     buf.extend(inst.bytes());
 }
 
+/// `SUB Xd, Xm, Xn` -> Subtract Xm and Xn and place the result into Xd.
+#[inline(always)]
+fn sub_reg64_reg64_reg64(
+    buf: &mut Vec<'_, u8>,
+    dst: AArch64GeneralReg,
+    src1: AArch64GeneralReg,
+    src2: AArch64GeneralReg,
+) {
+    let inst = ArithmeticShifted::new(true, false, ShiftType::LSL, 0, src2, src1, dst);
```

Comment:
Not for you to change in this PR (though would be something worth changing if you are interested), but I am realizing how unreadable these new methods are. They probably should either take records as inputs to add names `{op: .., s: ..., shifttype: ..., ..}`. That or at least change from bools to enums. So that the true and false at the beginning would have some meaning.